### ISSUE TITLE
DMSt Declaration not detected

### ISSUE THREAD

**Issue by frantzheld** [weglide/bugtracker#71]
**Describe the bug**
Declared DMSt Triangles/Squares are not scored as such.

**To Reproduce**
1. Go to https://beta.weglide.org/flights/10620
2. Click on DMSt
3. Scroll down to Scoring
4. See that only 827.29 points are scored (Triangle, see screenshot)

**Expected behavior**
The flight should be scored with 1140 DMSt points. (Declared)
(see https://www.onlinecontest.org/olc-3.0/gliding/flightinfo.html?dsId=7966916)
The flight of my team partner is scored correctly: https://beta.weglide.org/flights/1109

**Note:**
When I uploaded the flight for the first time (flight id 844), it was scored correctly.
Today I deleted the flight to test, if #69 can be applied.
After re-uploading the flight, this issue occured.

**Screenshots:**
![image](https://user-images.githubusercontent.com/47079209/104839307-6d8f3900-58c0-11eb-8cf4-a7e966a73a24.png)

**More examples:**
Another example is: https://beta.weglide.org/flights/6795

**Comment by moldhouse** [comment]
Thanks for reporting in such detail. Will look into it tomorrow. Just to identify potential error sources, have you played around with only declaration at all?

**Comment by frantzheld** [comment]
> Thanks for reporting in such detail. Will look into it tomorrow. Just to identify potential error sources, have you played around with only declaration at all?

If you mean _online_ declaration, the answer is yes. I've played around with the task planner and used the _Declare_ button.

**Comment by frantzheld** [comment]
> **Note:**
> When I uploaded the flight for the first time (flight id 844), it was scored correctly.
> Today I deleted the flight to test, if #69 can be applied.
> After re-uploading the flight, this issue occurred.

By the way: Deleting and re-uploading solved the #69 issue. The distance above the story pictures is correct now.

**Comment by moldhouse** [comment]
Okay I have identified the issue and fixed it. I will rescore your flight tomorrow. A task with 5WP can either be triangle with start on leg or rectangle and that's where we messed up.

Btw: We have opened up online declaration to past dates (which we are going to announce soon). This enables thorough testing of the feature. So having an active online declaration can shaddow the declaration in your file even for past flights.
